store/seed/test/10200__label.sql

```diff
@@ -0,0 +1,31 @@
+-- Label key for `bb.location`.
+INSERT INTO
```

Comment:
Shall we put this into init_schema.sql? since this is more like part of the schema (out builtin value)

Reply:
Yes, we should but in a separate CL once the feature is tested and ready for release.